<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondWindow</class>
 <widget class="QDialog" name="SecondWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_types">
        <attribute name="title">
         <string>Types</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QListWidget" name="listWidget"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_scope">
        <attribute name="title">
         <string>Scope</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QListWidget" name="listWidget_2"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_xaddr">
        <attribute name="title">
         <string>XAddr</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QListWidget" name="listWidget_3"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
